android/app/src/androidTest/java/com/trustwallet/core/app/blockchains/ark/TestArkTransactionSigning.kt

```diff
@@ -0,0 +1,47 @@
+package com.trustwallet.core.app.blockchains.ark
+
+import com.google.gson.Gson
+import com.google.gson.annotations.SerializedName
+import com.google.protobuf.ByteString
+import com.trustwallet.core.app.utils.toHexByteArray
+import junit.framework.Assert.assertEquals
+import org.junit.Test
+import wallet.core.jni.ARKSigner
+import wallet.core.jni.proto.ARK
+
+class TestArkTransactionSigning{
+    init {
+        System.loadLibrary("TrustWalletCore")
+    }
+    @Test
+    fun arkTransactionSigning(){
+        val signingInput = ARK.SigningInput.newBuilder()
+            .setAmount(1000000000)
+            .setPrivateKey(ByteString.copyFrom("d8839c2432bfd0a67ef10a804ba991eabba19f154a3d707917681d45822a5712"
```

Comment:
There's an extension method for conversion from ByteArray to ByteString.
Have a look in `utils/Extensions.kt`